examples/angular_bazel_architect/WORKSPACE

```diff
@@ -0,0 +1,39 @@
+# Bazel workspace created by @bazel/create 0.38.1
+# Declares that this directory is the root of a Bazel workspace.
+# See https://docs.bazel.build/versions/master/build-ref.html#workspace
+workspace(
+    # How this workspace would be referenced with absolute labels from another workspace
+    name = "angular_cli_demo",
+)
+
+# Install the nodejs "bootstrap" package
+# This provides the basic tools for running and packaging nodejs programs in Bazel
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "build_bazel_rules_nodejs",
```

Comment:
Nice! This is a great example.
Last few changes and I can merge this:
1) workspace name above should match the folder name `angular_bazel_architect`
2) rebase to master head & that should wipe out the root yarn.lock file diff as it was out-of-date
3) update the build_bazel_rules_nodejs http_archive here to 1.2.1 release